conf/messages.cy

```diff
@@ -248,23 +266,24 @@ emailAddress.error.required = Nodwch gyfeiriad e-bost
 emailAddress.error.length = Mae’n rhaid i’r cyfeiriad e-bost fod yn 85 o gymeriadau neu lai
 emailAddress.error.invalid = Nodwch gyfeiriad e-bost yn y fformat cywir, megis enw@enghraifft.com
 
-whomToPay.title = Pwy sydd i’w ad-dalu?
-whomToPay.heading = Pwy sydd i’w ad-dalu?
+whomToPay.title = Pwy ydych am i CThEM ad-dalu?
+whomToPay.heading = Pwy ydych am i CThEM ad-dalu?
 whomToPay.01 = Mewnforiwr
 whomToPay.02 = Cynrychiolydd
 whomToPay.checkYourAnswersLabel = Pwy sydd i’w ad-dalu?
-whomToPay.error.required = Dewiswch at bwy yr hoffech anfon yr ad-daliad
+whomToPay.error.required = Dewiswch pwy yr hoffech i CThEM ad-dalu
 
 repaymentType.title = Dewiswch ddull ad-dalu
 repaymentType.heading = Dewiswch ddull ad-dalu
 repaymentType.01 = Diwygiad mis cyfredol (CMA)
 repaymentType.02 = Trosglwyddiad banc (Bacs)
+repaymentType.hint.01 = Dim ond os gwnaethoch y taliad gwreiddiol gan ddefnyddio cyfrif gohirio tollau y gallwch ddefnyddio’r opsiwn hwn.
 repaymentType.checkYourAnswersLabel = Dull talu
 repaymentType.error.required = Dewiswch sut yr hoffech gael yr ad-daliad
 
 enterAgentEORI.title = Beth yw rhif EORI y mewnforiwr?
 enterAgentEORI.heading = Beth yw rhif EORI y mewnforiwr?
-enterAgentEORI.hint = Mae hyn yn 2 lythyren, wedi’u dilyn gan naill ai 12 neu 15 rhif, fel ‘GB123456789123’ neu ‘GB123456789123000’.
+enterAgentEORI.hint = Mae hwn yn cynnwys 2 lythyren, wedi’u dilyn gan naill ai 12 neu 15 rhif. Er enghraifft, ‘GB123456789123’ neu ‘GB123456789123000’.
```

Comment:
AMBER: We have removed the single quotes around the example in English.  Have the Welsh translation team accidentally added them back in?